test/scenarios/17. settings/advanced/show-hex-data.csv

```diff
@@ -0,0 +1,13 @@
+Step,Test Steps,Test Data,Expected Result,Notes
+1,Open the extension.,,The Welcome Back screen is shown,
+2,Proceed to Unlock the wallet.,password (8 characters min),,
+3,Click on the right corner setting(three dots) and click on the Settings from menu.,,Settings page appears,
+4,Click on the Advanced tab.,,Advanced settings page appears,
+5,Validate that the 'Show hex data' is default to off.,,The toggle is off by default,
+6,Toggle the Hex Data switch.,,The switch is turned on as expected,
+7,Click on the MetaMask logo,,Home page appears,
+7,Navigate to the Send transcation in the home menu.,,Send transcation appears,
```

Comment:
small typos: `transcation` should be `transaction

Reply:
Thanks Mariona, its corrected now.